setup.py

```diff
@@ -52,6 +52,7 @@ def read(path, encoding="utf-8"):
     "aenum>=2.2.0",
     "diskcache>=5.4.0",
     "retry>=0.9.2",
+    "psycopg2",
```

Comment:
Let us not make it a default requirement.

Reply:
Done